e2e-xvfb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+nohup /usr/bin/Xvfb :99 -ac -screen 0 1280x1024x24 &
```

Comment:
Just for reference, the projector for RH Summit has a 16:9 aspect ratio and 1280×720 resolution.